pkg/kn/commands/namespaced_test.go

```diff
@@ -95,6 +107,18 @@ func TestGetNamespaceDefaultAllNamespacesUnset(t *testing.T) {
 	if actualNamespace != expectedNamespace {
 		t.Fatalf("Incorrect namespace retrieved: %v, expected: %v", actualNamespace, expectedNamespace)
 	}
+
+	// Now do it again using -A instead
+	testCmd.SetArgs([]string{"-A"})
```

Comment:
Could we make those tests in a loop like in 

```golang
for _, arg := range []string { "--all-namespaces", "-A" } {
   ....

}
```

This would avoid copy & paste code. Same for the other test.

Reply:
sure

Reply:
done